### PR TITLE
running a mainnet node (modelled after running api node)

### PR DESCRIPTION
**Pull request by wileyj**
Update the mainnet docs to reflect the work done to run an api node. 
Essentially this is a copypasta of the api doc, with all references to postgres/api removed since we're only covering the stacks blockchain instance. 

of note: i did remove the part about running the container on a specific network to make it a little easier to run. for the API, this network is required for DNS - but for **only** the stacks blockchain, this is not essential. 

https://github.com/blockstack/docs/issues/1188

### CODE REVIEW

**Comment by wileyj** [comment]
Updated the testnet docs - will leave regtest docs as-is for now, until we figure out the future for that. 
Changes to taht doc should be in line with what was done for testnet, few text replacements should be all that's required

**Comment by wileyj** [comment]
@pgray-hiro

**Review by pgray-hiro — APPROVED** [review]
Approved with minor suggestions

File: src/pages/understand-stacks/running-mainnet-node.md [hunk 1]
```diff
@@ -13,239 +13,164 @@ images:
...
-INFO [1588108047.585] [src/chainstate/stacks/index/marf.rs:732] First-ever block 0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206
+```sh
+docker ps --filter name=stacks-blockchain
 ```
 
-## Running the mainnet node on Windows
+## Step 3: verifying the services
 
-### Prerequisites
+_Note: the initial burnchain header sync can take several minutes, until this is done the following commands will not work_
```
[the remark this answers is not in the document]
Comment (wileyj) [reply src/pages/understand-stacks/running-mainnet-node.md:119]: seems fine to me - i copied/pasted from the existing API docs for that (along with the linter)

**Pull request merged**